swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0, age = None):
+        super().__init__(category = "Clothing", condition = condition, age=age)
```

Comment:
Great job excluding `category` from the constructor parameters. We don't want to enable a caller to set a different category for a `Clothing` instance (though of course they could afterwards! Thanks Python!). But you're passing along the values we got from the Clothing constructor up to the Item constructor, so that it can perform any setup it needs, and it allows us to avoid writing duplicate code here too!